doc/api/fs.md

```diff
@@ -8,21 +8,25 @@
 
 <!-- source_link=lib/fs.js -->
 
-The `fs` module provides an API for interacting with the file system in a
-manner closely modeled around standard POSIX functions.
+The `fs` module allows interacting with the file system in a
+way closely modeled around standard POSIX functions.
 
 To use this module:
 
 ```js
 const fs = require('fs');
 ```
 
-All file system operations have synchronous and asynchronous forms.
+All file system operations have synchronous, callback, and Promise-based
+forms.
 
-The asynchronous form always takes a completion callback as its last argument.
-The arguments passed to the completion callback depend on the method, but the
-first argument is always reserved for an exception. If the operation was
-completed successfully, then the first argument will be `null` or `undefined`.
+## Callback Example
```

Comment:
nit: i think the flow of the documentation might be better with the examples shown in the order in which they were listed above (sync, callback, Promise).